src/types/utils/isLoadingOnyxValue.ts

```diff
@@ -0,0 +1,7 @@
+import type {OnyxKey, UseOnyxResult} from 'react-native-onyx';
+
+function isLoadingOnyxValue(...results: Array<UseOnyxResult<OnyxKey, unknown>[1]>): boolean {
```

Comment:
Also I'm wondering can we implement some kind of type narrowing to ensure that onyx values are defined? This way we could avoid optional chaining and nullish coalescing inside of components. Just an idea, we don't have to do it in this PR

Reply:
But the thing is that the Onyx value can still be `null` even if it's loaded. This helper function is useful for check if all values were loaded into cache and ready to be used, but it doesn't mean they are not `null`.

I previously thought about using type narrowing to consider the `status` off all the values to `loaded` if the condition is true, but not sure if it's really useful neither feasible.